### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by vivekparekh8**
This pull request adds a folder and file for UTDallas.

@crichID Your favorite food that should be added?
@Shobhit1 I can add yours as well.
@himanshusoni I can add yours as well.
🍔

### CODE REVIEW

File: vivekparekh8/vivekparekh8-githubteacher-utdallas.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# UTDallas
```
Comment (eminbagh) [inline vivekparekh8/vivekparekh8-githubteacher-utdallas.md:1]: How is the weather in Dallas?

File: vivekparekh8/vivekparekh8-githubteacher-utdallas.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Restaurants
```
Comment (Shobhit1) [inline vivekparekh8/vivekparekh8-githubteacher-utdallas.md:3]: you really want me to tell my favorite food? 🍔  😄  👍  @vivekparekh8

**Comment by vivekparekh8** [comment]
@Shobhit1 Is this ok ?

File: vivekparekh8/vivekparekh8-githubteacher-utdallas.md [hunk 3]
```diff
@@ -3,3 +3,5 @@
 ### Restaurants
 
 ### Things to do
+
+### Shobhit - Buffalo Wild wings, Madurai Tattukadai
```
Comment (Shobhit1) [inline vivekparekh8/vivekparekh8-githubteacher-utdallas.md:7]: Love buffalo wild wings, man.

**Comment by vivekparekh8** [comment]
#43 Done

**Pull request merged**